docs/index.rst

```diff
@@ -3,16 +3,185 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
-Ansible Builder
-===============
-
-Ansible Builder is a tool that aids in the creation of Execution Environments,
-which are container images that serve as Ansible :term:`control nodes<Control node>`.
+Introduction
+============
 
 Using Ansible content that depends on non-default dependencies can be tricky.
 Packages must be installed on each node, play nice with other software installed
 on the host system, and kept in sync.
 
+To help simplify this process, we have introduced the concept of Execution
+Environments, which you can create with Ansible Builder.
+
+.. contents::
+   :local:
+   :depth: 3
+
+
+Execution Environments
+----------------------
+
+Execution Environments are container images that serve as Ansible `control
+nodes`_. Starting in version 2.0, `ansible-runner`_ can make use of these images.
```

Comment:
Right here seems like the right place to add a link to the corresponding docs page for runner:

https://ansible-runner.readthedocs.io/en/latest/execution_environments.html

As you're adding a link to Ansible's docs page, it would make sense to cover this in the same change.

Reply:
Done